core/src/main/java/org/apache/gravitino/storage/relational/RelationalEntityStore.java

```diff
@@ -88,8 +89,12 @@ public void setSerDe(EntitySerDe entitySerDe) {
 
   @Override
   public <E extends Entity & HasIdentifier> List<E> list(
-      Namespace namespace, Class<E> type, Entity.EntityType entityType) throws IOException {
-    return backend.list(namespace, entityType);
+      Namespace namespace,
+      Class<E> type,
+      Entity.EntityType entityType,
+      List<Field> allowMissingFields)
```

Comment:
Is this design used for skipping some unnecessary fields? @yuqi1129 please also take a look about this design.

Reply:
Yes, because different upper layers may ignore different fields.  For example, list roles
Some upper layers may ignore objects and objects count.
Some upper layers may only ignore objects.
Some upper layers may not ignore any field.